rotkehlchen/exchanges/kraken.py

```diff
@@ -1149,3 +1160,27 @@ def query_kraken_ledgers(
                 return True  # we had errors so stop any further queries and quit
 
         return False  # no errors
+
+    def query_history_events(self) -> None:
+        self.msg_aggregator.add_message(
+            message_type=WSMessageType.HISTORY_EVENTS_STATUS,
+            data={
+                'status': str(HistoryEventsStep.QUERYING_EVENTS_STARTED),
+                'location': str(self.location),
+                'name': self.name,
+            },
+        )
+        with self.db.conn.read_ctx() as cursor:
+            self.query_kraken_ledgers(
```

Comment:
This will properly query only not queried parts of the events, right? It won't query entire history every time I hope ... otherwise this won't work for anybody. Add a comment to make it obvious.

Reply:
Yes, not all the events but only the parts that need it